code/modules/clothing/head/misc.dm

```diff
@@ -493,10 +493,69 @@
 
 /obj/item/clothing/head/dunce_cap
 	name = "dunce cap"
-	desc = "A conical paper hat which used to be used as a punishment in schools. Misbehaving children had to wear it while standing in a corner. The writing on it says \"DUNCE\"."
+	desc = "A conical paper hat which used to be used as a punishment in schools. Misbehaving children had to wear it while standing in a corner. The writing on it says \"DUNCE\". You feel a little bit stupider when you wear it."
 	icon_state = "duncecap"
 	item_state = "duncecap"
 
+/obj/item/clothing/head/dunce_cap/OnMobLife(var/mob/living/carbon/human/dunce)
+	..()
+	if(!istype(dunce))
+		return
+	if((dunce.get_item_by_slot(slot_head) == src))
+		if(dunce.getBrainLoss() < 20)
+			dunce.adjustBrainLoss(1)
+//Brain-damaged copypaste but this is thematic
+	if(prob(1))
```

Comment:
Absolutely unacceptable copypaste, put brain damage lines in a proc and call it instead.